ssv/spectest/tests/runner/full_happy_flow_sc.go

```diff
@@ -23,7 +23,7 @@ func fullHappyFlowSyncCommitteeContributionSC() *comparable.StateComparison {
 				StartingDuty: &testingutils.TestSyncCommitteeContributionConsensusData.Duty,
 				PreConsensusContainer: ssvcomparable.SetMessagesInContainer(
 					ssv.NewPartialSigContainer(3),
-					testingutils.SSVDecidingMsgsV(testingutils.TestSyncCommitteeContributionConsensusData, ks, types.BNRoleSyncCommitteeContribution)[:3]),
+					testingutils.ExpectedSSVDecidingMsgsV(testingutils.TestSyncCommitteeContributionConsensusData, ks, types.BNRoleSyncCommitteeContribution)[:3]),
```

Comment:
V function shouldn't recv version?

Reply:
same q for all below changes in the same file.

Reply:
The deprecated `SSVDecidingMsgs` method used hardcoded version.
Now the version used in the consensus data is passed to this method.

Since `SyncCommitteeContributionConsensusData` wasn't affected by the Capella fork we didn't create a factory for it that creates different variations.

Reply:
I still think its confusing that most V funcs recv a version and this one not. it felt like V is a convention and it's now broken.

Reply:
@olegshmuelov 
I kinda agree with @y0sher. Maybe we should quickly remove the V before merge?

Reply:
If we intend to remove it, we should follow these steps:
- Create a separate PR dedicated to the removal of the V functions, as it is not directly related to this PR.
- Keep in mind that this change will impact approximately ~120 files.
- It would be better to remove the V functions once all the tests using SC (State Comparison) functions have been updated. This approach ensures better code readability and clarity in determining if a test uses SC functions or not.
- In addition, there are certain versioned functions that have corresponding non-versioned counterparts. It is necessary to deprecate the non-versioned functions before proceeding with their removal.